docs/using-the-compiler.rst

```diff
@@ -181,7 +181,9 @@ at each version. Backward compatibility is not guaranteed between each version.
    - Introduces ``blobhash()`` in inline assembly and a corresponding global function to retrieve versioned hashes of blobs associated with the transaction (see `EIP-4844 <https://eips.ethereum.org/EIPS/eip-4844>`_).
    - Opcode ``mcopy`` is available in assembly (see `EIP-5656 <https://eips.ethereum.org/EIPS/eip-5656>`_).
    - Opcodes ``tstore`` and ``tload`` are available in assembly (see `EIP-1153 <https://eips.ethereum.org/EIPS/eip-1153>`_).
-- ``prague`` (**experimental**)
+- ``prague``
+- ``osaka`` (**experimental**)
+   - Introduces support to `EVM Object Format (EOFv1) <https://eips.ethereum.org/EIPS/eip-7692>`_.
```

Comment:
That's not really true until we merge #15828. But would probably make the most sense to only add it in #15824, together with the changelog entry, since until then it's not yet announced.